docs/advanced-features/compiler.md

```diff
@@ -30,19 +30,7 @@ We chose to build on SWC for a few reasons:
 - **WebAssembly:** Rust's support for WASM is essential for supporting all possible platforms and taking Next.js development everywhere.
 - **Community:** The Rust community and ecosystem are amazing and still growing.
 
-## Experimental Features
-
-### Minification
-
-You can opt-in to using the Next.js compiler for minification. This is 7x faster than Terser.
-
-```js
-// next.config.js
-
-module.exports = {
-  swcMinify: true,
-}
-```
+## Supported Features
 
 If you have feedback about `swcMinify`, please share it on the [feedback discussion](https://github.com/vercel/next.js/discussions/30237).
```

Comment:
This comment feels out of place in "Supported Features".

Reply:
Good catch. I originally moved the minifier since it was no longer under the experimental key, but moved it back since it is still just a release candidate and this got left behind.